server/sockets/list.js

```diff
@@ -329,24 +314,46 @@ const addListViewer = (io, dashboardClients, cohortClients) => data => {
     });
 };
 
-const setVote = socket =>
-  socket.on(ItemActionTypes.SET_VOTE_SUCCESS, data => {
-    const { listId, itemId } = data;
+const setVote = (io, listClients, viewersIds) => data => {
+  const { listId, itemId, userId } = data;
+
+  viewersIds.forEach(viewerId => {
+    const viewerIdAsString = viewerId.toString();
 
-    socket.broadcast
-      .to(listChannel(listId))
-      .emit(ItemActionTypes.SET_VOTE_SUCCESS, { listId, itemId });
+    if (viewerIdAsString !== userId.toString()) {
+      if (listClients.has(viewerIdAsString)) {
```

Comment:
check above comment

Reply:
https://github.com/sandstreamdev/eoc/pull/299#discussion_r319393737

Reply:
This check isn't complicated to implement, it's one additional if and you have all the needed data. I would check it but decide and resolve